.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name:  Which issue template do I choose?
+    url:   https://github.com/PolyhedralDev/Terra/wiki/How-To-Choose-An-Issue-Template
+    about: Click this if you don't know which issue template to select. This will help you make sure you choose the right one and provide enough information for us to help you.
+  - name:  Terra Wiki
```

Comment:
Add something along the lines of "If you have a basic support question, join the Discord instead."